python/ray/_private/external_storage.py

```diff
@@ -187,15 +187,14 @@ def _size_check(self, address_len, metadata_len, buffer_len, obtained_data_size)
         """Check whether or not the obtained_data_size is as expected.
 
```

Comment:
Do we plan to add `pydoclint` to pre-commit?

Reply:
Yep, just doing a fast-pass on a lot of the big errors before we do so the set of "exceptions" is small enough to be vaguely sensible